resources/deployments/deployment.go

```diff
@@ -80,6 +80,9 @@ type Deployment struct {
 	// Initialized with the "pending" counter set to total device count for deployment.
 	// Individual counter incremented/decremented according to device status updates.
 	Stats map[string]int `json:"-"`
+
+	// Total number of devices targeted
+	Count int `json:"device_count,omitempty"`
```

Comment:
This field is going to be stored in database, even if in current implementation is only alias for another query in response. Should be excluded from being stored  `bson:"-"`

Reply:
Call variable DeviceCount, Count is not self explanatory  on it's own.